install-go.md

```diff
@@ -62,6 +63,24 @@ mkdir -p $GOPATH/src $GOPATH/pkg $GOPATH/bin
 
 At this point you can _go get_ and the src/package/bin will be installed correctly in the appropriate $GOPATH/xxx directory.
 
+### Go Modules
+Go 1.11 introduced [Modules](https://github.com/golang/go/wiki/Modules), enabling an alternative workflow. This new approach will gradually [become the default](https://blog.golang.org/modules2019) mode, deprecating the use of GOPATH.
+
+Modules aim to solve problems related to dependency management, version selection and reproducible builds; they also enable users to run Go code outside of GOPATH.
+
+Using Modules is pretty straightforward. Just select any directory outside GOPATH as the root of your project, and create a new module with the go mod init command.
+```sh
+mkdir my-project
+cd my-project
+go mod init <modulename>
```

Comment:
It might be worth adding a more concrete example, how does a reader know what a `modulename` is?

Reply:
Hey quii. I added an example, let me know how it sounds. I hope it explained things a little better without making it too complicated.